hooks/sqs_configurator/package/secrets/kubernetes.go

```diff
@@ -0,0 +1,79 @@
+package secrets
+
+import (
+	"context"
+	"errors"
+	"go.uber.org/zap"
+	_ "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	coreV1Types "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"path/filepath"
+)
+
+var secretsClient coreV1Types.SecretInterface
+
+// KubeSecret holds the handler for the secret and auxiliary info
+type KubeSecret struct {
+	secret    *v1.Secret
+	namespace string
+}
+
+//New initializes the client with the given namespace and secret
+func New(namespace string, secretName string, isLocal *bool) (KubeSecret, error) {
+	var config *rest.Config
+	if *isLocal {
+		home, exists := os.LookupEnv("HOME")
+		if !exists {
+			home = "/root"
```

Comment:
/root?

Reply:
This is for the local setup

Reply:
takes root as home if home env is not set